src/Auctions/Auction.sol

```diff
@@ -137,10 +137,14 @@ contract Auction is Governance, ReentrancyGuard {
         require(_auctionLength < _auctionCooldown, "cooldown");
         require(_startingPrice != 0, "starting price");
 
+        // Cannot have more than 18 decimals.
+        uint256 decimals = ERC20(_want).decimals();
+        require(decimals <= WAD, "unsupported decimals");
```

Comment:
this should be 18, right? or alternatively `WAD / 10 ** decimals > 0`

Reply:
good catch